lab-judy/README.md

```diff
@@ -0,0 +1,24 @@
+Lab 11 Express api
+
+This is a project for Code 401 Day 11 using a simple resource API with express.
+
+This project takes data in the form of {name: <name>, age: <age.}
+
+Please install HTTP Pie for most effective use.
+
+Type "npm run start" in your terminal.
+
+In another terminal tab, use these commands:
+
+POST new data:
+
+echo '{"name": "<name>", "age": "<age>"}' | http:localhost:3000/api/person
+
+GET data:
+http GET localhost:3000/api/person id ==<id# here>
+
+UPDATE data:
+Completely forgot how to do this in the terminal, will get back to it later
```

Comment:
You should fix this! I won't take points off for the lab due yesterday, but you'll get a point knocked off for the lab turned in today. :) I'm pretty sure it's the same as the POST but you need to explicitly put PUT after the pipe character (and include a query string).